src/Cards.jsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+
+import Button from '@mui/material/Button';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+import Card from '@mui/material/Card';
+
+function Cards({
+  category, name, price, poster, cur,
+}) {
+  return (
+    <Grid
+      item
+      md="4"
+      sx={{
+        m: '10px 0 10px 0', backgroundColor: '#eceff1', display: 'flex', flexDirection: 'column', alignItems: 'center',
+      }}
+    >
+      <Card sx={{
+        display: 'flex',
+        boxShadow: 0,
+        flexDirection: 'column',
+        backgroundColor: '#eceff1',
```

Comment:
Would be nice to move all colors to pallette and use theme from theme

https://mui.com/material-ui/customization/palette/